src/storage/v2/disk/storage.cpp

```diff
@@ -1768,7 +1768,8 @@ utils::BasicResult<StorageManipulationError, void> DiskStorage::DiskAccessor::Co
     return StorageManipulationError{SerializationError{}};
   }
   spdlog::trace("rocksdb: Commit successful");
-
+  disk_storage->durable_metadata_.SaveBeforeClosingDB(disk_storage->timestamp_, disk_storage->vertex_count_,
+                                                      disk_storage->edge_count_);  // maybe rename this function
```

Comment:
This comment will be lost. Let's rename it or live with it :)
But do we close DB inside? I don't think so, also not in this function, so maybe call it differently? What about `UpdateMetadata`?